plugins/src/webrtcsink/imp.rs

```diff
@@ -2067,13 +1664,23 @@ impl WebRTCSink {
 
                 while interval.next().await.is_some() {
                     let element_clone = element_clone.clone();
-                    let peer_id_clone = peer_id_clone.clone();
                     if let (Some(webrtcbin), Some(element)) =
                         (webrtcbin.upgrade(), element_clone.upgrade())
                     {
-                        element
-                            .imp()
-                            .process_stats(&element, webrtcbin, &peer_id_clone);
+                            let peer_id = peer_id.to_string();
+                            let promise = gst::Promise::with_change_func(
+                                glib::clone!(@strong peer_id, @weak element => move |reply| {
```

Comment:
```suggestion
                                glib::clone!(@weak element => move |reply| {
```

Can just be moved in there or not?